src/charm.py

```diff
@@ -97,6 +98,13 @@ def __init__(self, *args: Any):
             extra_user_roles="SUPERUSER",
         )
 
+        self.ingress_per_unit = IngressPerUnitRequirer(
+            self,
+            "ingress",
```

Comment:
Just a minor nitpick:

We usually put constant integration names in the `constants.py`. It would be nice to follow that. Thank you.